.devcontainer/Dockerfile

```
@@ -5,7 +5,6 @@ ARG NODE_VERSION="none"
 RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
 ARG USER=vscode
 VOLUME /home/$USER/.m2
-VOLUME /home/$USER/.gradle
 ARG JAVA_VERSION=17.0.7-ms
-RUN sudo mkdir /home/$USER/.m2 /home/$USER/.gradle && sudo chown $USER:$USER /home/$USER/.m2 /home/$USER/.gradle
+RUN sudo mkdir /home/$USER/.m2 && sudo chown $USER:$USER /home/$USER/.m2
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)